README.md

```diff
@@ -94,7 +94,7 @@ to `Number.MIN_VALUE`.
 
     - `arguments` (*array*, optional) - An array of arguments to pass to the middleware factory.
 
-- `route` (*string*, *RegExp*, optional) - An express route against which the middleware should be registered. Please note that—if configuring `meddleware` with json files—you'll need to use something like [shortstop](https://github.com/krakenjs/shortstop) with [shortstop-regex](https://github.com/jasisk/shortstop-regex) to convert a string to RegExp.
+- `route` (*string*, *array*, *regexp*, optional) - An express route against which the middleware should be registered. Please note that—if configuring `meddleware` with json files—you'll need to use something like [shortstop](https://github.com/krakenjs/shortstop) with [shortstop-regex](https://github.com/jasisk/shortstop-regex) to convert a string to RegExp.
 
```

Comment:
Just following [the precedent](https://github.com/krakenjs/meddleware/blame/15a06158ae76016085eb1f17faa917aa4a251a81/README.md#L89). Can change both if you like?